setup.cfg

```diff
@@ -115,6 +115,7 @@ test =
     pytest-tinybird>=0.2.0
     aws-cdk-lib>=2.88.0
     websocket-client>=1.7.0
+    localstack-snapshot~=0.1.0
```

Comment:
Chiming in here from the side, after discussing this PR in https://github.com/localstack/localstack-snapshot/pull/2. 😛 
And unfortunately, I cannot comment on line 95, which is why I will just add two comments in one here:
- line # 95: Would be great to change this exact pin to a min version as discussed in https://github.com/localstack/localstack-snapshot/pull/2.
- line # 118: With the new dependency pinning we have a coordinated update mechanism, which automatically tries to update the lock files to the highest possible version. With #10195 we removed as many pins as possible to increase the efficiency of the automated updates. I think we should now avoid upper version limits wherever possible. It would be great if we could replace `~` with `>` in this line.

Reply:
Regarding the second point, I was mostly trying to avoid `>` due to the lib not strictly following semantic versioning yet. `rolo` had the same constraint, but since this was removed in the unpinning PR I guess we can remove it here as well.